docs/documentation/concepts/index.md

```diff
@@ -27,8 +27,8 @@ Things do not have to be physical devices; they can also represent a web service
 Things expose their capabilities through **Channels**.
 Whether an installation takes advantage of a particular capability reflected by a Channel depends on whether it has been configured to do so.
 When you configure your system, you do not necessarily have to use every capability offered by a Thing.
-You can find out what Channels are available for a Thing by looking at the documentation of the Thing's **Binding**.
-Bindings can be thought of as software adapters, making Things available to your home automation system.
```

Comment:
Imho this is a nice high-level introduction for a binding. This could have been the first sentence for the binding introduction.

Reply:
Good - I will either put the sentence back or I will keep the sense of it in what I write in response to your comments.